release/air_tests/air_benchmarks/mlperf-train/compute_cpu_16.yaml

```diff
@@ -6,6 +6,10 @@ max_workers: 0
 head_node_type:
     name: head_node
     instance_type: m5.4xlarge
+    BlockDeviceMappings:
+        - DeviceName: /dev/sda1
+          Ebs:
+              VolumeSize: 300
```

Comment:
Do you need to add this for the worker nodes as well?

Reply:
No, we use single node here. `max_workers` is set to 0.

Reply:
You may need to add the `DeleteOnTermination: true` unless the disk will be needed in the future.

Reply:
Ah I believe `DeleteOnTermination` defaults to `true` for root volume `/dev/sda1`: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/terminating-instances.html

Reply:
Added `DeleteOnTermination: true` to be more defensive anyway, thanks guys.

Reply:
Thanks for checking!